gcloud-java-compute/src/test/java/com/google/gcloud/compute/SnapshotTest.java

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+import static org.easymock.EasyMock.createMock;
+import static org.easymock.EasyMock.createStrictMock;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import com.google.common.collect.ImmutableList;
+
+import org.junit.Test;
+
+import java.util.List;
+
+public class SnapshotTest {
+
+  private static final String ID = "42";
+  private static final DiskId SOURCE_DISK = DiskId.of("project", "zone", "disk");
+  private static final Long CREATION_TIMESTAMP = 1453293540000L;
+  private static final String DESCRIPTION = "description";
+  private static final List<LicenseId> LICENSES = ImmutableList.of(
+      LicenseId.of("project", "license1"), LicenseId.of("project", "license2"));
+  private static final SnapshotId SNAPSHOT_ID = SnapshotId.of("project", "snapshot");
+  private static final SnapshotInfo.Status STATUS = SnapshotInfo.Status.CREATING;
+  private static final Long DISK_SIZE_GB = 42L;
+  private static final String SOURCE_DISK_ID = "diskId";
+  private static final Long STORAGE_BYTES = 24L;
+  private static final SnapshotInfo.StorageBytesStatus STORAGE_BYTES_STATUS =
+      SnapshotInfo.StorageBytesStatus.UP_TO_DATE;
+
+  private Compute serviceMockReturnsOptions = createStrictMock(Compute.class);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'serviceMockReturnsOptions' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2307619860/issues/source?bid=3140968&fileBranchId=3194371#l52)